README.md

```diff
@@ -155,6 +155,17 @@ cargo update -p package_name --precise 0.1.1
 
 </details>
 
+## Footprint
+
+The Xilem repository includes a lot of projects and examples, most of them pulling a lot of dependencies.
+
+If you contribute to Xilem on Linux or MacOS, we recommend using `split-debuginfo` in your `config.toml` file to reduce the size of the `target/` folder:
```

Comment:
```suggestion
If you contribute to Xilem on Linux or MacOS, we recommend using [`split-debuginfo`](https://doc.rust-lang.org/cargo/reference/profiles.html#split-debuginfo) in your
[`.cargo/config.toml`](https://doc.rust-lang.org/cargo/reference/config.html#hierarchical-structure) file to reduce the size of the `target/` folder:
```

It might also be worth mentioning that this is already the default on MacOS.

Reply:
The default on MacOS is "packed", which is not quite as good IMO.

Reply:
Oh. Fair enough - I only skimmed this!